test/Microsoft.ML.Functional.Tests/Explainability.cs

```diff
@@ -49,6 +51,43 @@ public void GlobalFeatureImportanceWithPermutationFeatureImportance()
                 Common.AssertMetricsStatistics(metricsStatistics);
         }
 
+        /// <summary>
+        /// GlobalFeatureImportance: PFI can be used to compute global feature importance. Here it is used with a model loaded from disk.
+        /// </summary>
+        [Fact]
```

Comment:
You could probably turn the above method into a `[Theory]` and then pass in `true` and `false` for a `bool saveModel` parameter. That way you can reuse all the code, and only change what you do with the model in between fitting and transforming the data.

Reply:
Thanks for the feedback. I didn't know about [Theory]'s. I've used them in my latest commit, in all the tests I had provided.